lib/rbi/parser.rb

```diff
@@ -429,7 +429,14 @@ def parse_sig(node)
     sig { params(node: AST::Node).returns(TEnumBlock) }
     def parse_enum(node)
       enum = TEnumBlock.new
-      node.children[2].children.each do |child|
+
+      if node.children[2].type == :begin
```

Comment:
There's a similar thing being done in the `T::Struct` parsing code: https://github.com/Shopify/rbi/blob/03d02e24f0dc768cca68aeb3f600dc209727a514/lib/rbi/parser.rb#L377-L381